crates/bevy_render/src/render_graph/base.rs

```diff
@@ -185,12 +223,145 @@ pub(crate) fn add_base_graph(config: &BaseRenderGraphConfig, world: &mut World)
         }
     }
 
+    if config.add_post_pass {
+        let main_render_texture_node = WindowTextureNode::new(
+            WindowId::primary(),
+            TextureDescriptor {
+                size: Extent3d::new(1, 1, 1),
+                mip_level_count: 1,
+                sample_count: msaa.samples,
+                dimension: TextureDimension::D2,
+                format: TextureFormat::Bgra8Unorm,
```

Comment:
What happens when a user wants a different buffer format?

Reply:
Fair question, that's one for the *how do we compose a rendergraph* discussion.